static/css/styles.css

```diff
@@ -383,6 +397,35 @@ li {
     margin-bottom: 0.5em;
 }
 
+/* Font Size and Weight Utility Classes */
+.fs-1 {

```

Comment:
Please don't override the Bootstrap font-sizes. They're global standard.
If needing to make a single element larger or smaller, then target that one specific element. Please remove the overrides for the `fs-#` classes and resubmit.

Reply:
I'm not sure that I did? Just added fs-6 as next size down? Removing that addition will resolve this convo?